api/v1beta1/consts.go

```diff
@@ -170,4 +170,6 @@ const (
 	AzureManagedControlPlaneKind = "AzureManagedControlPlane"
 	// AzureClusterIdentityKind indicates the kind of an AzureClusterIdentity.
 	AzureClusterIdentityKind = "AzureClusterIdentity"
+	// AzureNetworkPluginName is the name of the Azure network plugin.
+	AzureNetworkPluginName = "azure"
```

Comment:
nit: can we move this out of that const block, it's not a Kind so doesn't belong with the others

Reply:
can do in follow up